geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/list/AbstractRPopLPushIntegrationTest.java

```diff
@@ -99,6 +99,14 @@ public void rPopLPush_withNonexistentSourceKey_returnsNull() {
     assertThat(jedis.rpoplpush(SOURCE_KEY, DESTINATION_KEY)).isNull();
   }
 
+  @Test
+  public void rPopLPush_withNonexistentDestinationKey_returnsPoppedElement() {
```

Comment:
This test is duplicating "rPopLPush_createsListAtDestination_whenDestinationKeyIsEmpty" so should probably be removed.